extensions/ql-vscode/src/model-editor/modeled-method.ts

```diff
@@ -19,12 +19,87 @@ export type Provenance =
   // Entered by the user in the editor manually
   | "manual";
 
-export interface ModeledMethod extends MethodSignature {
-  readonly type: ModeledMethodType;
+export interface NoneModeledMethod extends MethodSignature {
+  readonly type: "none";
+  // Provenance is always propagated
+  readonly provenance: Provenance;
```

Comment:
I don't understand what this means that "provenance is always propagated". Could you explain a bit more? How can we have a provenance on a "none" modeling?

Reply:
This would essentially ensure that if you ran flow generation and then changed the type to unmodeled, and then changed it to something else, it would still show `df-manual`.

I actually think that doesn't make a ton of sense anymore, so I've removed this field. I believe that in a previous iteration it was somewhat harder to do, but there were very few places where we actually used this.